addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -182,7 +182,7 @@ options.registry.carousel = options.Class.extend({
         var $active = this.$inner.find('.item.active, .item.prev, .item.next').first();
         var index = $active.index();
         this.$('.carousel-control, .carousel-indicators').removeClass('hidden');
-        this.$indicators.append('<li data-target="#' + this.id + '" data-slide-to="' + cycle + '"></li>');
+        this.$indicators.append('<li data-target="#' + this.id + '" data-slide-to="' + cycle + '"></li> ');
```

Comment:
Add a comment or add the space in a separated call to `append` otherwise it is too easy to remove by mistake thinking it is useless.